testing/mjolnir/src/mjolnir_lib/mod.rs

```diff
@@ -78,3 +79,17 @@ pub fn build_monitor(progress_bar_mode: &ProgressBarMode) -> Monitor {
         ProgressBarMode::None => Monitor::Disabled(10),
     }
 }
+
+pub trait DiscriminationExtensions {
+    fn from_bool(testing: bool) -> Self;
```

Comment:
I'd be tempted to call this something like `from_testing_bool`, or something that describes what `true` means. If I saw `Discrimination::from_bool(true)`, I'm not sure it would be obvious whether it was prod or test 🤷